test/base/test_returns.py

```diff
@@ -58,15 +58,16 @@ def test_episodic_returns(size=2560):
             done=np.random.randint(100, size=size) == 0,
             rew=np.random.random(size),
         )
+
+        def vanilla():
+            return compute_episodic_return_base(batch, gamma=.1)
+
+        def optimized():
+            return fn(batch, gamma=.1)
+
         cnt = 3000
-        t = time.time()
-        for _ in range(cnt):
-            compute_episodic_return_base(batch, gamma=.1)
-        print(f'vanilla: {(time.time() - t) / cnt}')
-        t = time.time()
-        for _ in range(cnt):
-            fn(batch, None, gamma=.1, gae_lambda=1)
-        print(f'policy: {(time.time() - t) / cnt}')
+        print('GAE vanilla', timeit(vanilla, setup=vanilla, number=cnt))
+        print('GAE optim  ', timeit(optimized, setup=optimized, number=cnt))
```

Comment:
assert some speedup here?

Reply:
Numba optimized version is not the same thing as AsyncVenv, so I do not prefer to assert time here.